### DISCUSSION TITLE
part-ii/2021-123/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#391] [Question comments]
Part II 2021 Probability and Measure: Paper 3, Section II, $26 \mathrm{H}$ 

https://questions.tripos.org/part-ii/2021-123/

**Comment by ArthurConmy** [comment]
Bookwork. Then, because $L^2$ is complete, if $(X_i)$ converges then it converges to some $X \in L^2$. 

For the if direction $X = X_1+X_2+...$ works because $\mathbb{ E}[(X_n+X_{n+1}+...)^2] \to 0 $.

For the only if direction, suppose that $\mathbb{ E}[ | S_n - X |^2 ] \to 0$. Then we can check $X \in L^2$, so $X \in L^1$ too, and then 

$$ \mathbb{ E}[ (S_n - X)^2 ] \ge \mathbb{ E}[S_n^2] - 2\sqrt{ \mathbb{ E}[S_n^2] \mathbb{ E}[X^2] } + \mathbb{ E}[X^2] \to \infty $$

because $\mathbb{ E}[ S_n^2 ] \to \infty $.

Note that as stated in the examiner's report, the hint is useless.
